main.go

```diff
@@ -228,6 +236,12 @@ the "access" command instead.
 	Example: strings.Trim(`
   # List all secrets in the bucket "my-secrets"
   berglas list my-secrets
+
+  # List all secrets with names starting with "secret" in the bucket "my-secrets"
+  berglas list my-secrets --prefix secret
+
+  # List all generations of all secrets in the bucket "my-secrets"
+  berglas list my-secrets --generations
```

Comment:
perhaps `--all-generations` would be more clear despite being more verbose
because you're also defining a `--generation` flag

Reply:
Yeah I like that better, even though more verbose.